Content.Client/UserInterface/Screens/SeparatedChatGameScreen.xaml.cs

```diff
@@ -1,17 +1,36 @@
 using System.Numerics;
 using Content.Client.UserInterface.Systems.Chat.Widgets;
+using Content.Shared.CCVar;
 using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
 using Robust.Client.UserInterface.Controls;
 using Robust.Client.UserInterface.XAML;
+using Robust.Shared;
+using Robust.Shared.Configuration;
 
 namespace Content.Client.UserInterface.Screens;
 
 [GenerateTypedNameReferences]
 public sealed partial class SeparatedChatGameScreen : InGameScreen
 {
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] private readonly IUserInterfaceManager _uiManager = default!;
+
+    // The minimum width of InventoryGui + HotbarGui combined.
+    // Used to keep the PDA from overlapping the shoes slot.
+    private const int INVENTORY_MINIMUM_WIDTH =
+        (125 /* width */ - 2 /* negative margin */) * 2 /* ItemStatusPanel */
+        + 68 /* width */ * 10 /* ItemSlotButton, including inventory */
+        + 67 /* width */ * 2 /* hands */;
```

Comment:
These values are bound to fall out of line so you can't just rely on magic numbers like this.